README.md

```diff
@@ -2,44 +2,203 @@
+regressions that our human expertise missed so that we can fix them right away.
+(Call it "continual integration" rather than "continuous integration".) -->
+
+
+## Driving Ursabot
+
+Allowing PR reviewers to request additional checks on demand within the review
+process makes it easier for us to apply extra scrutiny at review time while
+also conserving CI bandwidth by using human expertise to know which checks are
+needed.
+
+### via Comments
+
+Ursabot receives github events through a webhook. It listens on pull request
+comments mentioning @ursabot. It follows the semantics of a command line
+interface, to see the available commands add a comment on the pull request: `@ursabot --help`.
+
+The @ursabot GitHub user will respond or [react][github-reactions] that it has
+started a build for you. Unfortunately, it does not currently report back
+on the build status. The reporters are already implemented. They will be
+enabled once the proper github integration permissions are set for the
+[apache/arrow][arrow-repo] repository. Until that you have to search around the
+[buildbot UI][ursabot-url] for it. The command parser is implemented in [commands.py](commands.py).
+
+Currently available commands:
+
+  - `@ursabot build`: Triggers all the ursabot tests. These tests are run
+    automatically, but this is a convinient way to force a re-build.
+  - `@ursabot benchmark`: Triggers C++ benchmarks and sends back the results as
+    a github comment and highlights the regressions.
+  - `@ursabot crossbow test cpp-python`: Triggers the `cpp-python` test group
+    defined in [test.yml][crossbow-tests] and responds with a URL pointing to submitted crossbow branches at the github UI showing the build statuses.
+  - `@ursabot crossbow package wheel conda`: Triggers the `wheel` and `conda`
+    crossbow packaging groups defined in [tasks.yml][crossbow-tasks].
+
+Note that the commands won't trigger any builds if the commit message contains
+a skip pattern, like `[skip ci]` or `[ci skip]`.
+
+### via the Web UI
+
+You can also initiate a build for a specific architecture/configuration in the
+[buildbot UI][ursabot-url]. Navigate to [Builds > Builders][ursabot-builders],
+select a builder, and click `Build apache/arrow` buttin at the top right. This triggers the force schedulers where you can specify a branch and/or commit to
+build. In the future specialized builders will have different fields to provide
+the neccessary information.
+
+### via CLI
+
+Buildbot supports submitting local patches directly to the cluster and
+triggering specific builders. The `TryScheduler` is a really handy way to test
+local changes without polluting the git history:
+
+```bash
+buildbot try \
+  --connect=pb \
+  --master=... \
+  --username=... \
+  --passwd=... \
+  --get-builder-names
+```
+
+If someone wants to use this feature then please raise an issue, because it requires custom credentials.
+
+## Running Ursabot
```

Comment:
This is how it is running on ci.ursalabs.org? I.e. this is a guide not for the user of ursabot but for a sysadmin? (If that's right, be clear about that, as in, we don't want to imply that to use ursabot, you have to serve it locally.)